src/__tests__/linter.test.ts

```diff
@@ -41,20 +41,18 @@ describe('linter', () => {
 
   beforeEach(() => {
     spectral = new Spectral();
-    spectral.registerFormat('oas3', isOpenApiv3);
     spectral.registerFormat('oas2', isOpenApiv2);
+    spectral.registerFormat('oas3', isOpenApiv3);
   });
 
-  test('should not lint if passed in value is not an object', async () => {
+  test('should not throw if passed in value is not an object', () => {
     const fakeLintingFunction = jest.fn();
     spectral.setFunctions({
       [fnName]: fakeLintingFunction,
     });
     spectral.setRules(rules);
 
-    const result = await spectral.run('123');
-
-    expect(result).toHaveLength(0);
+    return expect(spectral.run('123', { ignoreUnknownFormat: true })).resolves.toBeTruthy();
```

Comment:
Do we need to talk about ignoreUnknownFormat in here? Seems like it's more about strings vs objects? 

Maybe it could be two different tests, one to prove the default and another further down where we do more testing of ignoreUnknownFormat?

Reply:
Yeah, good point.